cleanlab/experimental/tf_nlp.py

```diff
@@ -0,0 +1,129 @@
+"""
+A Scikeras classifier (adapted for huggingface models) which
+can be used for finding label issues in any text datasets.
+"""
+
+from typing import Dict
+from scikeras.wrappers import KerasClassifier
+from sklearn.preprocessing import FunctionTransformer
+import tensorflow as tf
+import numpy as np
+
+
+class HuggingfaceKerasClassifier(KerasClassifier):
+    def __init__(self, train_input: Dict, seq_len: int, **kwargs):
+        """
+        Basic scikeras/scikeras does not directly provide the option of having
+        a multi-input model, i.e the input must be in the form (num_samples, feature_size). However, you can work around this problem by
+        adding the `feature_encoder` property to the class which extends
+        Scikitlearn's BaseEstimator.
+
+        Example of use:
```

Comment:
This is overall an awesome contribution!  If this code happens to identify interesting label issues in the IMDB dataset (you can try `lnl.find_label_issues()` to quickly find out), then might be interested in submitting your example and the label issues you found in IMDB as a guest post for the HuggingFace blog?  https://huggingface.co/blog

I think they might be interested in featuring that!  Feel free to chat with me in our [Slack channel](https://join.slack.com/t/cleanlab-community/shared_invite/zt-17lszn4hv-gg2FhZPXYfljq_l01uo92g) if you're interested in discussing details.